backend/routers/index.js

```diff
@@ -1,5 +1,6 @@
 const troops = require('./troops');
+const buildings = require('./buildings');
 
 module.exports = {
-  troops,
+  troops, buildings,
```

Comment:
```suggestion
  troops, 
  buildings,
```
we usually list the exported modules in new lines